lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -60,6 +60,15 @@ div.course-wrapper {
       }
     }
 
+    .sequential-status-message {
+      margin-bottom: $baseline;
+      background-color: $gray-l4;
+      color: $black-t2;
+      padding: ($baseline/4) ($baseline/2);
+      margin-top: -($baseline);
+      @include font-size(13);
+    }
```

Comment:
After reviewing these changes visually on the sandbox, I have a few more changes suggested here prior to merging this PR.

* remove the color rule
* remove the margin-top rule
* padding --> ($baseline*0.75)
* background-color --> $gray-l5; 
* add border-radius rule set to 3px.  

cc'ed @asadiqbal08